.github/workflows/ci.yml

```diff
@@ -59,3 +59,43 @@ jobs:
         name: ${{ matrix.os }}-screenshots
         # Directory containing files to upload
         path: screenshots
+
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    # Platforms to build on/for
+    strategy:
+      max-parallel: 1
+      matrix:
+        os:
+        - windows-latest
+        # - ubuntu-latest
+        # - macos-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Node.js, NPM and Yarn
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - name: Build Electron app
+      shell: bash
+      env:
+        GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
+        GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NODE_OPTIONS: --max-old-space-size=4096
+        SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY }}
+        SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      run: |
+        yarn
+        yarn build --publish never
+    - name: Upload artifact
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        # Artifact name
+        name: ${{ matrix.os }}-artifact
+        # Directory containing files to upload
+        path: dist_electron
```

Comment:
On PR, it will build the app only on `linux` (before it was built same as for Release, except it was not published as a Github Release).